src/components/App.tsx

```diff
@@ -3420,6 +3436,10 @@ class App extends React.Component<AppProps, AppState> {
             pointerDownState.origin.y,
           );
 
+        if (pointerDownState.hit.element?.locked && !event.metaKey) {
```

Comment:
what's the purpose of `!event.metaKey`?

Also, what does this case handle at all? `hit.element` will be null if element locked in the first place